LIPs/LIP-36.md

```diff
@@ -112,6 +120,11 @@ Update the earnings claiming algorithm to:
 
 Any read only functions used to calculate a delegator's stake and fees including unclaimed earnings (i.e. `BondingManager.pendingStake()` and `BondingManager.pendingFees()` will need to be updated to follow the above logic (without any storage updates such as zeroing out the orchestrator's cumulative values).
 
+#### setLIPUpgradeRound(uint256 _LIP, uint256 _round)
+
+sets the key in the `LIP_UPGRADE_ROUNDS` mapping to `_LIP`.
+The `_round` argument is optional and will be set to  `currentRound+1` if `_round == 0`.
```

Comment:
Allowing the `_round` argument to be optional doesn't seem that helpful since the caller would still need to specify a value (i.e. 0) and if that's the case it seems just as easy to provide a specific round number. If we can simplify this function to only be responsible for setting a storage value and nothing else that would be ideal.

Reply:
fixed in 4823679